configure.ac

```diff
@@ -1117,17 +1117,27 @@ if test "X$HDF5_DOXYGEN" = "Xyes"; then
   AC_SUBST([DOXYGEN_MACRO_EXPANSION])
   AC_SUBST([DOXYGEN_OUTPUT_DIRECTORY])
   AC_SUBST([DOXYGEN_EXAMPLES_DIRECTORY])
+  AC_SUBST([DOXYGEN_LAYOUT_FILE])
+  AC_SUBST([DOXYGEN_HTML_HEADER])
+  AC_SUBST([DOXYGEN_HTML_FOOTER])
+  AC_SUBST([DOXYGEN_HTML_EXTRA_STYLESHEET])
+  AC_SUBST([DOXYGEN_HTML_EXTRA_FILES])
 
   DOXYGEN_PACKAGE=${PACKAGE_NAME}
   DOXYGEN_VERSION_STRING=${PACKAGE_VERSION}
   DOXYGEN_INCLUDE_ALIASES='$(SRCDIR)/doxygen/aliases'
   DOXYGEN_PROJECT_LOGO='$(SRCDIR)/doxygen/img/HDFG-logo.png'
-  DOXYGEN_PROJECT_BRIEF="C-API Reference"
+  DOXYGEN_PROJECT_BRIEF=
   DOXYGEN_INPUT_DIRECTORY='$(SRCDIR) $(SRCDIR)/doxygen/dox'
   DOXYGEN_OPTIMIZE_OUTPUT_FOR_C=YES
   DOXYGEN_MACRO_EXPANSION=YES
   DOXYGEN_OUTPUT_DIRECTORY=hdf5lib_docs
   DOXYGEN_EXAMPLES_DIRECTORY='$(SRCDIR)/doxygen/examples'
+  DOXYGEN_LAYOUT_FILE=hdf5doxy_layout.xml
+  DOXYGEN_HTML_HEADER=hdf5_header.html
+  DOXYGEN_HTML_FOOTER=hdf5_footer.html
+  DOXYGEN_HTML_EXTRA_STYLESHEET=hdf5doxy.css
+  DOXYGEN_HTML_EXTRA_FILES='hdf5_navtree_hacks.js img/ftv2node.png img/ftv2pnode.png'
```

Comment:
Should these use absolute paths with `SRCDIR` to support out-of-tree builds? When I run the standard

```bash
mkdir build
cd build
/path/to/hdf5/configure --enable-doxygen
make doxygen
```
Doxygen can't find these files.

Reply:
Good point. Let me fix this.